src/adapter.js

```diff
@@ -173,12 +173,11 @@ var createMochaReporterConstructor = function (tc, pathname) {
   }
 }
 /* eslint-disable no-unused-vars */
-var createMochaStartFn = function (mocha) {
+var createMochaStartFn = function (karma, mocha) {
   /* eslint-enable no-unused-vars */
-  return function (config) {
-    var clientArguments
-    config = config || {}
-    clientArguments = config.args
+  return function () {
```

Comment:
Could there be existing callers in other code that expect the argument to be used?